### PR TITLE
Make script to update supported hypersync/rpc urls automatically

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: docs/HyperSync/hypersync-url-endpoints.md [hunk 1]
```diff
@@ -7,61 +7,65 @@ slug: /hypersync-url-endpoints
...
+| Optimism Sepolia | 11155420 | https://optimism-sepolia.hypersync.xyz or https://11155420.hypersync.xyz | gold |   |
+| Polygon | 137 | https://polygon.hypersync.xyz or https://137.hypersync.xyz | gold |   |
+| Polygon Amoy | 80002 | https://polygon-amoy.hypersync.xyz or https://80002.hypersync.xyz | bronze |   |
+| Polygon Zkevm | 1101 | https://polygon-zkevm.hypersync.xyz or https://1101.hypersync.xyz | gold |   |
+| Publicgoods | 424 | https://publicgoods.hypersync.xyz or https://424.hypersync.xyz | bronze |   |
+| Rsk | 30 | https://rsk.hypersync.xyz or https://30.hypersync.xyz | bronze |   |
+| Scroll | 534352 | https://scroll.hypersync.xyz or https://534352.hypersync.xyz | gold |   |
+| Sepolia | 11155111 | https://sepolia.hypersync.xyz or https://11155111.hypersync.xyz | gold |   |
+| Shimmer Evm | 148 | https://shimmer-evm.hypersync.xyz or https://148.hypersync.xyz | bronze |   |
+| Staging Eth | 1 | https://staging-eth.hypersync.xyz or https://1.hypersync.xyz | gold |   |
```
Comment (SvenMuller95) [inline docs/HyperSync/hypersync-url-endpoints.md:59]: What is Staging Eth

**Review by SvenMuller95 — APPROVED** [review]
i've adjusted it in my contribution commit, but would be nice to reference the networks as they are called.
e.g. Zksync -> ZKsync 
e.g. Polygon Zkevm -> Polygon zkEVM

I've also changed these pages to be called "Supported Networks" instead of "URL endpoints". Our users are more likely to search "supported networks" "supported chains" etc. not "URL endpoint"

File: docs/HyperSync/hypersync-url-endpoints.md (hunk 1 above)
Comment (JasoonS) [reply docs/HyperSync/hypersync-url-endpoints.md:63]: Yeah, I'll filter it out :grimacing: It is Ozgurs test/staging instance :+1:

**Comment by JasoonS** [comment]
I'll show you how to edit that :+1:

File: scripts/update-endpoints.js [hunk 2]
```diff
@@ -3,9 +3,13 @@ const fs = require('fs');
 const url = 'https://chains.hyperquery.xyz/active_chains';
 const renameConfig = {
   eth: 'Ethereum Mainnet',
+  "polygon-zkevm": "Polygon zkEVM",
+  "zksync": "ZKsync",
```
Comment (JasoonS) [inline scripts/update-endpoints.js:9]: @SvenMuller95 - this renames the chains how we want :+1:

File: scripts/update-endpoints.js [hunk 3]
```diff
@@ -3,9 +3,13 @@ const fs = require('fs');
...
   // Add other renaming rules here
 };
 
+const filterEndpoints = [/^staging-/];
```
Comment (JasoonS) [inline scripts/update-endpoints.js:13]: This regular expression is used to filter out any chain starting with `staging-` (such as staging-eth

**Comment by JasoonS** [comment]
Will fix this:
image

**Pull request merged**